test/tests/samplers/base/tests

```diff
@@ -151,15 +151,15 @@
       input = errors.i
       max_parallel = 1 # the number of local samples changes with number of cores
       expect_err = "The number of entries in the DenseMatrix \(1115098112\) exceeds the allowed limit of 429496729."
-      cli_args = "Samplers/sample/num_cols=1e9 UserObjects/test/test_type=getLocalSamples"
+      cli_args = "Samplers/sample/num_cols=1000000000 UserObjects/test/test_type=getLocalSamples"
 
       detail = "if the number of entries in the local sample matrix exceeds the maximum allowed, and"
     []
     [getNextLocalRow_max]
       type = RunException
       input = errors.i
       expect_err = "The number of entries in the std::vector \(1000000000\) exceeds the allowed limit of 429496729."
-      cli_args = "Samplers/sample/num_cols=1e9 UserObjects/test/test_type=getNextLocalRow"
+      cli_args = "Samplers/sample/num_cols=1000000000 UserObjects/test/test_type=getNextLocalRow"
```

Comment:
Does 1e9 not work?

Reply:
It doesn't seem too, this is an exploratory change.

Reply:
```
field node 'Samplers/sample/num_cols' does not hold an int-typed value (val='1e9')
```